docs/user-guide/widgets.md

```diff
@@ -90,13 +112,33 @@ Just below the chart's preview, the user is allowed to set:
 
 * The widget **Description**
 
-!!! note
+!!! Note
     None of these options are mandatory, you can add the widget to the map without filling in these fields.
```

Comment:
None of these fields are mandatory, it is possible to save/add the widget to the map without filling them.